src/components/entity/ha-entity-lock.html

```diff
@@ -0,0 +1,142 @@
+<link rel="import" href="../../../bower_components/polymer/polymer-element.html">
+<link rel="import" href="../../../bower_components/paper-icon-button/paper-icon-button.html">
+
+<dom-module id="ha-entity-lock">
+  <template>
+    <style>
+      :host {
+        white-space: nowrap;
+      }
+
+      paper-icon-button {
+        color: var(--primary-text-color);
+        transition: color .5s;
+      }
+
+      paper-icon-button[state-active] {
+        color: var(--primary-color);
+      }
+
+      paper-toggle-button {
+        cursor: pointer;
+        --paper-toggle-button-label-spacing: 0;
+        padding: 13px 5px;
+        margin: -4px -5px;
+      }
+    </style>
+
+    <paper-button-group>
```

Comment:
I'm not sure you need paper-button-group here. If you do - please import it.

Reply:
Yes I removed it already.